packages/blockchain-extension/extension/commands/packageSmartContractCommand.ts

```diff
@@ -105,12 +105,9 @@ export async function packageSmartContract(workspace?: vscode.WorkspaceFolder, o
             const pkgFile: string = path.join(resolvedPkgDir, `${properties.workspacePackageName}@${properties.workspacePackageVersion}.tar.gz`);
             const pkgFileExists: boolean = await fs.pathExists(pkgFile);
             if (pkgFileExists) {
-                if (language === 'golang') {
-                    throw new Error('Package with name and version already exists. Please input a different name or version for your Go project.');
-                } else if (language === 'java') {
-                    throw new Error('Package with name and version already exists. Please input a different name or version for your Java project.');
-                } else {
-                    throw new Error('Package with name and version already exists. Please change the name and/or the version of the project in your package.json file.');
+                const warnOverwrite: vscode.MessageItem = await vscode.window.showWarningMessage(`${properties.workspacePackageName}@${properties.workspacePackageVersion}.tar.gz already exists. Would you like to replace the existing package?`, { title: 'No' }, { title: 'Yes' });
+                if (!warnOverwrite || warnOverwrite.title === 'Cancel') {
```

Comment:
In your popup, users can either select 'Yes' or 'No'.
In this if statement, you check if they selected 'Cancel' - this needs to change (or the options presented!)